src/js/components/stock-movement-wizard/outboundImport/subsections/OutboundImportBasicDetails.jsx

```diff
@@ -20,6 +21,17 @@ const OutboundImportBasicDetails = ({ control, errors }) => {
     minSearchLength: state.session.searchConfig.minSearchLength,
   }));
 
+  const loadOutboundLocations = debounceLocationsFetch(
+    debounceTime,
+    minSearchLength,
+    null,
+    false,
+    false,
+    true,
+    false,
+    StockMovementDirection.OUTBOUND,
+  );
```

Comment:
I know this looks ugly and all of the values between `minSearchLength` and `direction` are the default values.
At first I thought to extract them from a destructed object but there would be a little to much work to change each instance of `debounceLocationsFetch` which is called accross the whole application.

Maybe we'll do it some other day :wink:

Reply:
@awalkowiak what do you think about creating a tech debt ticket for moving all of the `debonuceXYZFetch` to the hook for debounce fetching? We can also refactor those methods to take an object as an argument instead of having those ugly positional arguments.

Reply:
@alannadolny sounds good